components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -76,6 +77,7 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
 
     this.serviceGroupStatus$ = this.store.select(serviceGroupStatus);
     this.serviceGroups$ = this.store.select(allServiceGroups);
+    this.serviceGroupHealthSummary$ = this.store.select(allServiceGroupHealth);
 
```

Comment:
This PR is ready to merge!!! Good work @Shadae 

As a tiny suggestion if you would like to hook up the Service Groups Total to the pagination you could just add this like and it would magically work!!! 

Or @kmacgugan can do it in a separate PR, up to you two to decide! 
```suggestion
    this.serviceGroupHealthSummary$.subscribe((sgHealthSummary) => {
      this.totalServiceGroups = sgHealthSummary['total']
    });
```